doc/posts/zitification/README.md

```diff
@@ -0,0 +1,18 @@
+# Zitification
+
+"Zitification" is the act of taking an application and incorporating a Ziti SDK into that application. Once an
+application has a Ziti SDK incorporated into it, that application can now access network resources securely from
+anywhere in the world provided that the computer has internet access: NO VPN NEEDED, NO ADDITIONAL SOFTWARE NEEDED.
+
+Simply integrating the Ziti SDK into your application and enrolling the application itself into a Ziti Network provides
+you with _tremendous_ additional security. By using the Ziti SDK to access remote resources your application will
+become _IMMUNE_ to classic [ransomware attacks](https://netfoundry.io/ztna-ransomware/) of land, expand/multiply,
+destroy. When your application uses a Ziti Network configure with a truly zero-trust mindset it will be impervious to
+the "expand/multiple" phases of a classic [ransomware attacks](https://netfoundry. io/ztna-ransomware/). As recent
```

Comment:
Space in URL `https://netfoundry. io/ztna-ransomware`

Reply:
fixed